kubemarine/core/utils.py

```diff
@@ -486,6 +486,35 @@ def _test_version(version: str, numbers_amount: int) -> List[int]:
     raise ValueError(f'Incorrect version \"{version}\" format, expected version pattern is \"{expected_pattern}\"')
 
 
+def parse_aligned_table(table_text: str) -> List[Dict[str, str]]:
+    """
+    Parse aligned table into the list of {header: cell} map.
+    The text with table can be initially produced, for example, by https://pkg.go.dev/text/tabwriter.
+
+    :param table_text: aligned table as string
+    :return: List of rows
+    """
+    rows = table_text.strip().split('\n')
+    headers = rows[0].split()
+    headers_num = len(headers)
+    headers_pos = [rows[0].index(h) for h in headers]
```

Comment:
Do you guarantee, that no header can be substring for another one?

Reply:
Thank you for pointing out this gap. Although this code much relies on format, for example it assumes that there is no spaces in header names, this particular gap can be fixed. Now both header name and positions are detected in one pass.